### PR TITLE
Update: electerm.electerm version 1.14.0

### PR DESCRIPTION
**Pull request by DasCanard**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21734)

### CODE REVIEW

File: manifests/e/electerm/electerm/1.14.0/electerm.electerm.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- WSL
+- Developer-Tools
+- Utilities
+- cli
+- cmd
+- ps
+- terminal
+- File manager
+- sftp
+- ssh
```
Comment (ItzLevvie) [inline manifests/e/electerm/electerm/1.14.0/electerm.electerm.locale.en-US.yaml:36]:
```suggestion
- console
- command-line
- shell
- command-prompt
- powershell
- wsl
- developer-tools
- utilities
- cli
- cmd
- ps
- terminal
- file-manager
- sftp
- ssh
```

File: manifests/e/electerm/electerm/1.14.0/electerm.electerm.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+PackageVersion: 1.14.0
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+  Scope: machine
```
Comment (ItzLevvie) [inline manifests/e/electerm/electerm/1.14.0/electerm.electerm.installer.yaml:14]:
```suggestion
  Scope: user
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/e/electerm/electerm/1.14.0/electerm.electerm.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+- interactive
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+  Scope: user
+  InstallerUrl: https://github.com/electerm/electerm/releases/download/v1.14.0/electerm-1.14.0-win-x64-installer.exe
+  InstallerSha256: 7FEE2B43E381A7C6CF3B433505E805C7BD40DFB9EC2D86E453093916D8EE18A0
+  UpgradeBehavior: install
```
Comment (OfficialEsco) [inline manifests/e/electerm/electerm/1.14.0/electerm.electerm.installer.yaml:17]:
```suggestion
- InstallerLocale: en-US
  Architecture: x64
  InstallerType: nullsoft
  Scope: user
  InstallerUrl: https://github.com/electerm/electerm/releases/download/v1.14.0/electerm-1.14.0-win-x64-installer.exe
  InstallerSha256: 7FEE2B43E381A7C6CF3B433505E805C7BD40DFB9EC2D86E453093916D8EE18A0
  InstallerSwitches:
    Custom: /CURRENTUSER
  UpgradeBehavior: install
- InstallerLocale: en-US
  Architecture: x64
  InstallerType: nullsoft
  Scope: machine
  InstallerUrl: https://github.com/electerm/electerm/releases/download/v1.14.0/electerm-1.14.0-win-x64-installer.exe
  InstallerSha256: 7FEE2B43E381A7C6CF3B433505E805C7BD40DFB9EC2D86E453093916D8EE18A0
  InstallerSwitches:
    Custom: /ALLUSERS
  UpgradeBehavior: install
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Needed for compatibility

**Review by OfficialEsco — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**